### PR TITLE
Renamed and Updated CI scripts to include custom configurations

### PR DESCRIPTION
**Pull request by fgarci03**
Renamed scripts and changed 'shippable' to 'ci'
Updated test_ci_backend to include a custom .ini file with CI settings
Added .gitignore

### CODE REVIEW

File: test_ci_backend.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+# python static analysis
+prospector --profile-path=. --profile=.prospector.yml --path=src --ignore-patterns=static --output-format=xunit > shippable/testresults/python.xml
```
Comment (pedrorodriguesgomes) [inline test_ci_backend.sh:6]: rename to ci/testresults/python.xml

File: test_ci_backend.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+# python tests
+py.test -c pytest_ci.ini --cov=src --cov-report=xml tests/
```
Comment (pedrorodriguesgomes) [inline test_ci_backend.sh:9]: provide example of pytest_ci.ini

File: test_ci_backend.sh [hunk 3]
```diff
@@ -3,10 +3,18 @@
 set -e
 
 # python static analysis
-prospector --profile-path=. --profile=.prospector.yml --path=src --ignore-patterns=static --output-format=xunit > shippable/testresults/python.xml
```
Comment (lfrodrigues) [inline test_ci_backend.sh]: has to be shippable as this is folder shippable expects

File: test_ci_frontend.sh [hunk 4]
```diff
@@ -3,10 +3,10 @@
 set -e
 
 # javascript static analysis
-./node_modules/.bin/eslint -c .eslintrc -f junit src/static tests/js > shippable/testresults/eslint.xml
```
Comment (lfrodrigues) [inline test_ci_frontend.sh:6]: again folder must be shippable

File: test_ci_mobileapp.sh [hunk 5]
```diff
@@ -3,4 +3,4 @@
 set -e
 
 # javascript static analysis
-./node_modules/.bin/eslint -c .eslintrc -f junit js tests/js > shippable/testresults/eslint.xml
```
Comment (lfrodrigues) [inline test_ci_mobileapp.sh:6]: same

File: test_pipelines.sh [hunk 6]
```diff
@@ -5,7 +5,7 @@ BackendTests () {
 }
 
 FrontendTests () {
-    ./node_modules/.bin/istanbul cover --report cobertura --dir shippable/codecoverage node_modules/mocha/bin/_mocha -- --require tests/require --recursive --compilers js:babel-core/register --reporter mocha-junit-reporter tests/js/
```
Comment (lfrodrigues) [inline test_pipelines.sh:8]: no idea how pipelines handles this. @DfilipeS?

**Pull request merged**